Framework/Algorithms/src/ElasticWindow.cpp

```diff
@@ -202,6 +202,8 @@ void ElasticWindow::exec() {
     convUnitQ2->execute();
     outputQSquared = convUnitQ2->getProperty("OutputWorkspace");
   }
+  auto yLabel = outputQSquared->YUnitLabel();
+  outputQSquared->setYUnitLabel("ln(" + yLabel + ")");
```

Comment:
```suggestion
  outputQSquared->setYUnitLabel("\ln(" + yLabel + ")");
```
we should be able to use Latex

Reply:
I don't think that works when trying to give it in a cpp string.

Reply:
that makes me sad